tests/zero_code_change/test_tensorflow2_integration.py

```diff
@@ -117,6 +117,9 @@ def helper_test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
 
         hook = smd.get_hook()
         assert hook
+        # Check if the hook was executed with the default
+        # hook configuration
+        assert hook.has_default_hook_configuration()
```

Comment:
isn't this applicable to the gradtape integration test too?

Reply:
Yes, will add the check there as welll